### PR TITLE
Nodemailer #330

### LINKED ISSUE #330: NodeMailer
## 🚀 Feature

By this you can use node.js to mail from one person to another person

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

yes

## Motivation

previous project motivated me to do this

## Pitch

Since it is a node project ,it  should be added in website Repo , It is almost important in each and every website

### PR DESCRIPTION
**Pull request by mayank-del**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

This is nodemailer by which you can mail a form having message to anyone, when i was beginner , i suffered alot to learn to this .So, i am sharing this to open source.

Fixes #330

### Checklist

- [ y] I've read the contribution guidelines.
- [y ] I've checked the issue list 
- [ y] I've edited the `README.md` and link to my code.

No issues present in this work.
## screenshots of the output

Gmail

Screenshot 2022-03-10 212659

### CODE REVIEW

**Comment by photon149** [comment]
@mayank-del Please check the issue and files changed.They are for ping pong game

**Comment by mayank-del** [comment]
Very sorry for that, I will fix it today

**Comment by mayank-del** [comment]
@photon149  Please check  it once, I have sent nodemailer now.

**Review by photon149 — CHANGES_REQUESTED** [review]
@mayank-del Please attach a README file , alongside output ss/

**Comment by mayank-del** [comment]
sir see I added Screenshots  of output in description at the top and added Readme.md in the recent commit as well

**Comment by photon149** [comment]
@mayank-del Add the output SS in your README.md file as well.

**Comment by mayank-del** [comment]
@photon149 , please check it , I have copied the screenshot to readme.md as well

**Comment by photon149** [comment]
Great . Approved from my end.
@Kushal997-das @prakhar3062 have a look.

**Review by prakhar3062 — APPROVED** [review]
okay but you added two project node mailer and pingpong in same pr

**Comment by mayank-del** [comment]
@prakhar3062 sir, i think nodemailer is at correct position , i have to change directory of pingpong game, can u suggest any different directory

File: Web Development/Basic/NodeMailer/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+       then inside mailOption object, In "from" key, you have to add the same email as value.
+
+       then do nodemon app.js in terminal to start th server, run localhost:3000 in your browser to see it.
+
+       But one more thing make sure to enable SMTP MAIL WITH YOUR SENDER GMAIL, else google might gonna stop your email and declare you unauthorised.
+       
+       
+  Output Screenshots    
+
+![Screenshot 2022-03-10 212659](https://user-images.githubusercontent.com/82238106/158013570-f79d29b5-89f1-4839-bf1a-9f634ac4b296.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/NodeMailer/README.md:20]: Reduce the image size .

Comment (Kushal997-das) [inline Web Development/Basic/NodeMailer/README.md:20]: Reduce the image size.

File: Web Development/Basic/NodeMailer/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Gmail recieved to the reciever
+
+
+![Gmail](https://user-images.githubusercontent.com/82238106/158013610-f834af1c-5937-4657-aaa5-c49295321b16.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/NodeMailer/README.md:25]: Reduce the image size.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@mayank-del Do the required changes ASAP

**Comment by mayank-del** [comment]
@Kushal997-das  ,Sir i fixed everything , now please accept the pull request

**Review by Kushal997-das — COMMENTED** [review]
@mayank-del You didn't change anything so by the end of this day(EOD) change it or else we gonna close this PR .

**Comment by mayank-del** [comment]
@Kushal997-das  sir you told to to reduce  screenshot size only , i did it,. now, please merge it.

**Comment by mayank-del** [comment]
@Kushal997-das  , please check , screenshot size reduced

**Comment by mayank-del** [comment]
@Kushal997-das  sir , kindly check it and inform me, please

File: Web Development/Basic/NodeMailer/README.md (hunk 1 above)
Comment (Kushal997-das) [reply Web Development/Basic/NodeMailer/README.md:20]: Reduce the image size from here only means from`readme.md` file

**Comment by Kushal997-das** [comment]
> @Kushal997-das sir , kindly check it and inform me, please

**Comment by Kushal997-das** [comment]
@mayank-del reduce the image size from`readme.md` file don't you understand ?

**Comment by mayank-del** [comment]
@Kushal997-das , I changed the size of img in both readme.md  as well as above in comment, please check and merge .

File: Web Development/Basic/NodeMailer/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+       But one more thing make sure to enable SMTP MAIL WITH YOUR SENDER GMAIL, else google might gonna stop your email and declare you unauthorised.
+       
+       
+  Output Screenshots    
+
+
+![Gmail](https://user-images.githubusercontent.com/82238106/159414410-24848cbe-e8c1-4e27-aa2f-add53127bbdf.jpg)
+
+![Screenshot 2022-03-10 212659](https://user-images.githubusercontent.com/82238106/159414446-808e24fb-d79b-41a3-9d73-ccba413f92b8.jpg)
+![Gmail](https://user-images.githubusercontent.com/82238106/159414755-b1c2909c-9090-4da8-ae0f-5fdc0dbdd5a3.jpg)
```
Comment (Kushal997-das) [inline Web Development/Basic/NodeMailer/README.md:24]: Use this format to reduce the size .

``

Comment (mayank-del) [reply Web Development/Basic/NodeMailer/README.md:24]: I reduced the size to 29kb ,it is 1/3 of original.

How we can apply html tags in readme.md??

Comment (photon149) [reply Web Development/Basic/NodeMailer/README.md:24]: @mayank-del by reducing the size he meant the size of the images , i.e. height and width in the README file ,  not the file size in KB

**Comment by mayank-del** [comment]
now check @photon149  sir, i decrease dimension to 50%

**Comment by mayank-del** [comment]
@photon149 , @Kushal997-das  please it

**Review by Kushal997-das — APPROVED** [review]
@mayank-del LGTM 💯

**Comment by Kushal997-das** [comment]
@photon149 Have a look !

**Comment by photon149** [comment]
Looks fine to me.
@Kushal997-das It's good to merge .

**Comment by mayank-del** [comment]
@photon149  yes yes ,it is good to merge

**Review by photon149 — APPROVED** [review]
Approved from my end.
@Kushal997-das

**Pull request merged**